civis/io/_tables.py

```diff
@@ -618,7 +615,9 @@ def dataframe_to_civis(df, database, table, api_key=None, client=None,
                        diststyle=None, distkey=None,
                        sortkey1=None, sortkey2=None,
                        headers=None, credential_id=None,
-                       polling_interval=None,
+                       primary_keys=None, last_modified_keys=None,
+                       execution="immediate",
+                       delimiter=None, polling_interval=None,
                        archive=False, hidden=True, **kwargs):
     """Upload a `pandas` `DataFrame` into a Civis table.
 
```

Comment:
I can't add a comment to a hidden part of the PR for some reason, but the docs for `existing_table_rows` should have `upsert` as an option as well

Reply:
Good catch. I will update this.